sql/core/src/main/java/org/apache/spark/sql/sources/v2/vectorized/ColumnarBatch.java

```diff
@@ -87,19 +79,7 @@ public void remove() {
   }
 
   /**
-   * Resets the batch for writing.
-   */
-  public void reset() {
```

Comment:
remove this as it's for `WritableColumnVector` only

Reply:
+1